modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -34,51 +28,228 @@ components:
                     type: object
                 externalReferenceCode:
                     description:
-                        The asset library's site external reference code.
-                    readOnly: true
+                        The asset library's external reference code.
                     type: string
                 id:
                     description:
-                        The asset library's ID.
+                        The asset library's site ID.
                     format: int64
                     readOnly: true
                     type: integer
-                linkedSiteIds:
+                logo:
+                    description:
+                        The asset library's logo.
+                    type: string
+                name:
+                    description:
+                        The asset library's name.
+                    type: string
+                name_i18n:
+                    additionalProperties:
+                        type: string
+                    description:
+                        The localized asset library's name.
+                    type: object
+                numberOfUserAccounts:
```

Comment:
Are we going to show the number of users and userGroups? I remember that we talked about it and said it could lead to performance issues (might be wrong)